### DISCUSSION TITLE
requestPointerLock() does not work in a webview

### DISCUSSION THREAD

**Discussion by TwitchBronBron** [microsoft/vscode-discussions#55] [Extension Development QnA] [answered]
I opened a github issue on the vscode repository about this, but wanted to raise visibility a bit. Does anyone know how to get `requestPointerLock()` to work on an element in a custom webview in vscode? (more details can be found in the attached link)

https://github.com/microsoft/vscode/issues/154198

**Comment by isidorn** [comment, marked as answer]
I see bot assigned that issue to @mjbvz in the VS Code repo, I expect him to respond since he is the right person.
And I will mark this question as answered since the right place to discuss it is on the VS Code side. Thanks!
